tensorboard/plugins/audio/BUILD

```diff
@@ -45,6 +45,27 @@ py_test(
     ],
 )
 
+py_test(
+    name = "audio_plugin_test_notf",
+    size = "small",
+    srcs = ["audio_plugin_test.py"],
+    main = "audio_plugin_test.py",
+    srcs_version = "PY2AND3",
+    deps = [
+        ":audio_plugin",
+        ":summary",
+        "//tensorboard:expect_numpy_installed",
+        "//tensorboard:expect_tensorflow_installed",
```

Comment:
Given that this target depends on `no_tensorflow`, why does it also
depend on `expect_tensorflow_installed`?

Reply:
So, the tricky thing here (and with other plugin tests) is that they use TensorFlow to construct test data, etc. but we want to test the plugin itself without TensorFlow. The `no_tensorflow` accounts for this by forcing TensorBoard to use the stub implementation even when TensorFlow is installed, which is what we want for these tests. For now this seemed like the best option?

Reply:
It would be better to just prune away the TensorFlow dependencies from these tests and then skip any that fundamentally require TF using something like `unittest.skipUnless(USING_REAL_TF)`, but I grant that this is not a small task, so I've created an issue to track it here: https://github.com/tensorflow/tensorboard/issues/2007

Can you just add a TODO above each of these tests like this:
`TODO(#2007): Remove this after pruning unnecessary TensorFlow deps in main test`

Reply:
Can you actually put the comment above the entire `py_test()`?  I should have been clearer - the whole test should be removed after the regular test (with `notf` in the name) can be run without a tensorflow dependency.